opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -59,7 +57,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 spanBuilder.asChildOf(extractedSpanContext);
             }
 
-            Span span = spanBuilder.start();
+            Span span = spanBuilder.startManual();
+            tracer.makeActive(span);
```

Comment:
@objectiser if span is deactivated in request filter then during processing of request (before response filter) span will not be picked up as parent span. In such case there is no sense to activate/deactivate span. 
For example:
1. Send request
2. Request Endpoint send another REST request. Here we should get active span from request filter to use as parent for new REST request.
3. Send response